bigbluebutton-html5/tests/webdriverio/pageobjects/landing.page.js

```diff
@@ -16,25 +16,45 @@ class LandingPage extends Page {
     return 'http://localhost:8080/demo/demoHTML5.jsp';
```

Comment:
This shouldn't be fix in code, the host name should come from some config, from the `package.json` or test-config.json, since the URL can vary from user to user, same for port.

We can add the following snipped to the `package.json`
```
 "scripts": {
    .....
  },
 "config": {
    "host":"localhost:8080"
  },
  "repository": {
    .......
  }
```
And dynamic get in code using `process.env.npm_package_config_host`

`return '${process.env.npm_package_config_host}/demo/demoHTML5.jsp';`